samples/benchmarks/coremark/src/main.c

```diff
@@ -170,8 +172,20 @@ static int button_init(void)
 	return ret;
 }
 
+static void main_thread_priority_cooperative_set(void)
```

Comment:
Function name is a bit confusing as the function actually updates priority of the current thread (not necessarily `main` thread). Maybe we could just simply call `k_thread_priority_set(k_current_get(), K_PRIO_COOP(2));` in `main` function?

Reply:
I would not insist (as it would not break runtime)